### ISSUE TITLE
Checkout session missing the `created` key

### ISSUE THREAD

**Issue by arlyon** [stripe/stripe-mock#371]
Hello

I noticed one of our tests in arlyon/async-stirpe started failing recently, as stripe-mock doesn't return the `created` key as expected in checkout sessions. This was added as a non-optional field in v206.

See here:

https://github.com/stripe/openapi/commit/7450237cc1dee0f3578456b7e12dc374e1b77921#diff-eb42f9b1394184cdc66c6edf3cdba135114b3279ee6ea981fe7b5eb7fd6e50b4R4022-R4026

**Comment by remi-stripe** [comment]
@arlyon Thanks for the report! We'll look into fixing this!

**Comment by pakrym-stripe** [comment]
@arlyon thank you for the report. We've released an updated version of stripe-mock (0.145.0) and I've confirmed that `created` property is returned.

```bash
> curl http://localhost:12111/v1/checkout/sessions/123   -u sk_test_: | grep created
  "created": 1234567890,
```

**Comment by arlyon** [comment]
Amazing, cheers for the quick turnaround! :)
